backend/helper_zero/views/auth.py

```diff
@@ -1,13 +1,20 @@
+import os
 from rest_framework import status, viewsets
 from rest_framework.response import Response
+from google.oauth2 import id_token
```

Comment:
This looks like we need to `pip install` something; update `Pipfile` with the dep?

Reply:
yup good catch!